packages/notifications/push/amplify_push_notifications/ios/Classes/AmplifyPushNotificationsPlugin+AppDelegate.swift

```diff
@@ -115,4 +115,11 @@ extension AmplifyPushNotificationsPlugin {
     public func applicationDidBecomeActive(_ application: UIApplication) {
         application.registerForRemoteNotifications()
     }
+
+    public func applicationDidEnterBackground(_ application: UIApplication) {
+        // When App enters background we remove the cached laucnhNotification
+        // as when the App reopens after this point, there won't be a noticiation
```

Comment:
Typos
```suggestion
        // When App enters background we remove the cached launchNotification
        // as when the App reopens after this point, there won't be a notification
```

Reply:
The 2nd commit is the fix and relevant test.
The 1st commit is the iOS unit test.